specification/consumption/resource-manager/Microsoft.Consumption/preview/2019-05-01-preview/consumption.json

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
Please format this json file

Reply:
Done, please let me know if it looks good now.